src/components/nc-modal.vue

```diff
@@ -128,13 +128,13 @@ export default {
 
     resizeModal() {
       this.isDesktopDevice = this.getDesktopDevice()
-      if (!this.isDesktopDevice) {
+      if (this.fixedSize || this.isDesktopDevice) {
+        this.heightByDevice = this.height
+        this.widthByDevice = this.width
+      } else {
         const padding = parseInt(this.padding) * 2
         this.widthByDevice = `calc(100vw - ${padding}px)`
```

Comment:
He visto que en el código a la hora de usar esta prop se usa `padding="0px"` esto va a resultar en:
```
calc(100vw - 0pxpx)
```

Habría que revisar o confirmar como se va a usar esta prop

Reply:
Está bien, si te fijas `padding` es una constante que se define en la linea superior, el calculo no se hace con this.padding.